backend/src/main/scala/cromwell/backend/standard/callcaching/StandardCacheHitCopyingActor.scala

```diff
@@ -209,10 +223,14 @@ abstract class StandardCacheHitCopyingActor(val standardParams: StandardCacheHit
 
   when(FailedState) {
     case Event(f: IoReadForbiddenFailure[_], Some(data)) =>
-      handleForbidden(
+      handleBlacklistingForForbidden(
         path = f.forbiddenPath,
         andThen = stayOrStopInFailedState(f, data)
```

Comment:
Hmm, TOL (just checking...) is this `stayOrStopInFailedState(f, data)` evaluated before or after it gets passed in to `handleBlacklistingForForbidden`?

Reply:
yes